test/combineReducers.spec.js

```diff
@@ -237,5 +237,26 @@ describe('Utils', () => {
 
       spy.restore()
     })
+
+    it('only warns for unexpected keys once', () => {
```

Comment:
Do I undestand correctly that tests now depend on global shared state and run order? This doesn’t look very good.